### ISSUE TITLE
Tiktxk seems broken, videos get an error

### ISSUE THREAD

**Issue by smiba** [Britmoji/tiktxk#2]
Hi,

As of yesterday all videos seem to get an error instead:

```
{

    "_comment": "If you're here, and have no idea what this nerd shit is, something went wrong when trying to embed a TikTok video. We don't support slideshows or LIVE videos, and we can't show videos that are private, or have been deleted. Sorry! Something wrong? Open an issue on GitHub at https://tiktxk.com/issue <3",
    "error": "Unexpected end of JSON input",
    "success": false

}

```

Maybe tiktok changed the way they return data?

**Comment by KodingDev** [comment]
Appears so, we use an internal endpoint for fetching data recently since it allows us to fetch data on slideshows. I assume the endpoint is failing because one of the two signatures has expired or something in our request.

(If anyone wants to try and reproduce the issue how we do, here's a curl)

<details>
<summary>Request</summary>

```
curl --request POST \
  --url 'https://api19-normal-c-alisg.tiktokv.com/aweme/v1/multi/aweme/detail/?device_id=7150415238323324421&channel=googleplay&aid=1233&app_name=musical_ly&version_code=260403&version_name=26.4.3&device_platform=android&device_type=Pixel%2B5&os_version=12' \
  --header 'content-type: application/x-www-form-urlencoded' \
  --header 'user-agent: com.zhiliaoapp.musically/2022604030 (Linux; U; Android 12; en_AU; Pixel 5; Build/SQ3A.220705.003.A1;tt-ok/3.12.13.1)' \
  --header 'x-argus: I/Qq8Aucfjg1Fs2fRb1R/JfVq5LxV9BTvBcIDOTlk6e2btnd9hkNZjPtQg6zHlM1T6bnp3H0xOV6JEoF+nDCU9hqVm7ZuFlUEkQkxoxgRUd0kCZEy2gkecaTzI2J5EjdOx30q1O+qg0+qFG3X0krNo25U27nZtAHLufJxpK54SqAxFmuJwc2E8/GgZT0iV3sCiqA3cTiRsU04Mm9w1e4ZS0HEFCz6wGR6BuqvQCQiwJwxEmmowbmYCdXrykhbdyuGSAcmO4tbXPhCODwrrcgADX/ooAVcsP60SOUj0FdCwNrZumH7k/27Xwz2xIv2iTQ7vmsnfflh+ZEtq7/jElUDCIb' \
  --header 'x-ladon: QcAZOBZPYsUdPhgSa32MfkUyMIs4EEAJQelTPIIyIReyUK1K' \
  --data 'aweme_ids=[7149771205250174250]' \
  --data request_source=0
```

</details>

Might just fallback to the actual TikTok API until we can figure out how to generate the `x-argus` and `x-ladon` headers ourselves.

**Comment by KodingDev** [comment]
Yeah so update on this, got everything working as normal again (minus slideshows but they were a mid implementation regardless) but for some reason all the video URLs respond with 403's.

Example: https://tiktxk.com/meta/7158119889994075398/video

Any ideas are appreciated, but that's all I've got tonight since it's 5AM as I write this lmao.

**Comment by KodingDev** [comment]
Fixed as of [ee666b4](https://github.com/Britmoji/tiktxk/commit/ee666b42aa2c5b51f05df9d1b4c273f1e7d4bce1), thanks for the report :)
